packages/ripple-nuxt-tide/README.md

```diff
@@ -67,7 +244,133 @@ You will need below config file in your Nuxt root to make @dpc-sdp/ripple-nuxt-t
 
 ### Custom theming
```

Comment:
This is a structure of our `create-sdp-app` template, so should we doc there.

Reply:
Where is the `create-sdp-app` template? Need some direction for this one too.

Reply:
Doesn't exist yet - Will be added in https://digital-engagement.atlassian.net/browse/SDPA-2709 - POC exists here https://github.com/dpc-sdp/ripple-monorepo-poc/tree/monorepo-poc/packages/create-ripple-app